src/lib/validators.ts

```diff
@@ -1,28 +1,31 @@
-'use strict';
 import * as Bluebird from 'bluebird';
 import { stat } from 'fs';
 import { isFQDN, isPort } from 'validator';
 
 const statAsync = Bluebird.promisify(stat);
 
+/**
+ * Validate a directory exists
+ * @param {string} path
+ * @param {boolean} raise
+ * @returns {Promise<boolean>}
+ */
 export async function validateDirectoryPath(
 	path: string,
 	raise = true,
 ): Promise<boolean> {
 	try {
 		const stat = await statAsync(path);
 		if (!stat.isDirectory()) {
-			if (raise) {
-				throw new Error('Error: ' + path + ' is not a directory');
-			}
-			return false;
+			throw new Error('Error: ' + path + ' is not a directory');
```

Comment:
TypedError.

Reply:
Validators are dropped, as we are now using schema everywhere.